.circleci/orbs/helix-post-deploy/orb.yml

```diff
@@ -157,22 +163,30 @@ commands:
               fi
               # use package version as action version
               actionVersion=`node -e "console.log(require('./package.json').version.match(/^[0-9]+/)[0])"`
+              actionStatus="/_status_check/healthcheck.json"
               if [ "${nrURL}" == "" ]; then
-                # assemble runtime url
-                nrURL="https://adobeioruntime.net/api/v1/web/${actionNS}/${actionPackage}/${actionName}@v${actionVersion}/_status_check/healthcheck.json"
+                # assemble universal runtime url
+                nrURL="https://helix-pages.anywhere.run/${actionPackage}/${actionName}@v${actionVersion}${actionStatus}"
```

Comment:
I'd probably turn `helix-pages.anywhere.run` into a parameter, so that the orb stays applicable to other projects, but make it the default value, so that we don't have to change anything in our configs.